fabric-data-generation-api-v1/src/main/java/net/fabricmc/fabric/api/datagen/v1/provider/FabricRecipesProvider.java

```diff
@@ -16,64 +16,17 @@
 
 package net.fabricmc.fabric.api.datagen.v1.provider;
 
-import java.nio.file.Path;
-import java.util.Set;
-import java.util.function.Consumer;
-
-import com.google.common.collect.Sets;
-import com.google.gson.JsonObject;
-
-import net.minecraft.data.DataCache;
-import net.minecraft.data.server.RecipesProvider;
-import net.minecraft.data.server.recipe.RecipeJsonProvider;
-import net.minecraft.data.server.recipe.ShapedRecipeJsonFactory;
-import net.minecraft.data.server.recipe.ShapelessRecipeJsonFactory;
-import net.minecraft.util.Identifier;
-
 import net.fabricmc.fabric.api.datagen.v1.FabricDataGenerator;
 
 /**
- * Extend this class and implement {@link FabricRecipesProvider#generateRecipes}.
- *
- * <p>Register an instance of the class with {@link FabricDataGenerator#addProvider} in a {@link net.fabricmc.fabric.api.datagen.v1.DataGeneratorEntrypoint}
+ * @deprecated use {@link FabricRecipeProvider} instead.
  */
-public abstract class FabricRecipesProvider extends RecipesProvider {
+@Deprecated
+public abstract class FabricRecipesProvider extends FabricRecipeProvider {
 	protected final FabricDataGenerator dataGenerator;
 
-	public FabricRecipesProvider(FabricDataGenerator dataGenerator) {
+	protected FabricRecipesProvider(FabricDataGenerator dataGenerator) {
```

Comment:
Why is the access modifier for this constructor changed here?

Reply:
Not sure, must have been changed by another commit and reverted by this one - will fix.